### ISSUE TITLE
RNIapModule.buyItemByType got 8 arguments, expected 9

### ISSUE THREAD

**Issue by ADAustin-SE** [dooboolab/react-native-iap#1180]
### Version of react-native-iap
5.1.1
### Version of react-native
63.2
### Platforms you faced the error (IOS or Android or both?)
Android
### Expected behavior
Expected a successful subscription. I upgraded from 4.5.4 since upgrading subscription tiers wasn't working with Google since it required the past purchase token, but now subscriptions as a whole stopped functioning.
### Actual behavior
I got this error upon running requestSubscription(***product_id***):
![Screenshot_1604700870](https://user-images.githubusercontent.com/19857157/98420136-211bb780-204c-11eb-8908-f74d354d8eff.png)

### Tested environment (Emulator? Real Device?)
Tested on Emulator and Real Device, same result in both environments
### Steps to reproduce the behavior
Upgraded react-native-iap in my package.json to the latest version, cleared my node_modules, cache, package.json-lock, npm installed, and then ran through a normal attempt on a subscription. It retrieved my products from google successfully, and opened a connection, just the purchase attempt that causes the above.

**Comment by abhishektiwari-kiwi** [comment]
How to resolve this?

**Comment by marf** [comment]
Same problem here.
@ADAustin-SE have you got a solution?

**Comment by ADAustin-SE** [comment]
@abhishektiwari-kiwi @marf My fix is slightly unknown, I cleared everything multiple times, npm cache clean --force, removed the package-lock.json, then ran a new npm install and I still had the same issue. 

I attempted a subscription again a few hours later, and it worked, I hadn't made any changes to my dependencies in that time. So, yeah, something is definitely cached that is messing it up, just waiting to be cleared. I didn't find out what it was, it corrected itself, sorry.
